src/EnergyPlus/DataSurfaces.cc

```diff
@@ -86,6 +94,14 @@ namespace DataSurfaces {
 	using namespace DataPrecisionGlobals;
 	using namespace DataVectorTypes;
 	using namespace DataBSDFWindow;
+  using namespace DataHeatBalance;
```

Comment:
Then we should fix wiki documentation:
https://github.com/NREL/EnergyPlus/wiki/Coding-Standards-Part-1-Style
First it states "Style guidelines are not overly strict." and then later "Tabs are not allowed, and a mixture of tabs and spaces is strictly forbidden".